pyproject.toml

```diff
@@ -1,6 +1,150 @@
+[project]
+name = "composeml"
+readme = "README.md"
+description = "a framework for automated prediction engineering"
+dynamic = ["version"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development",
+    "Topic :: Scientific/Engineering",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Operating System :: OS Independent",
+]
+authors = [
+    {name="Alteryx, Inc.", email="open_source_support@alteryx.com"}
+]
+maintainers = [
+    {name="Alteryx, Inc.", email="open_source_support@alteryx.com"}
+]
+keywords = ["prediction engineering", "data science", "machine learning"]
+license = {text = "BSD 3-clause"}
+requires-python = ">=3.8,<4"
+dependencies = [
+    "pandas >= 1.3.0",
+    "tqdm >= 4.32.0",
+    "matplotlib >= 3.3.3",
+    "seaborn >= 0.11.0",
+]
+
+[project.urls]
+"Documentation" = "https://compose.alteryx.com"
+"Source Code"= "https://github.com/alteryx/compose/"
+"Changes" = "https://compose.alteryx.com/en/latest/release_notes.html"
+"Issue Tracker" = "https://github.com/alteryx/compose/issues"
+"Twitter" = "https://twitter.com/alteryxoss"
+"Chat" = "https://join.slack.com/t/alteryx-oss/shared_invite/zt-182tyvuxv-NzIn6eiCEf8TBziuKp0bNA"
+
+[project.optional-dependencies]
+test = [
+    "pip >= 21.3.1",
+    "pytest-cov == 3.0.0",
+    "pytest-xdist >= 2.5.0",
+    "wheel >= 0.33.1",
+    "featuretools >= 1.4.0",
+    "woodwork >= 0.11.0",
+    "pyarrow >= 3.0.0",
+]
+updater = [
+    "alteryx-open-src-update-checker >= 2.1.0"
+]
+docs = [
+    "evalml >= 0.45.0"
+]
+dev = [
```

Comment:
Previously the dev requirements also included the test requirements. Should we also add `compose[test]` in this section?